test/CrowdsaleGenTest.js

```diff
@@ -174,44 +177,45 @@ contract('LifCrowdsale Property-based test', function(accounts) {
 
     await runGeneratedCrowdsaleAndCommands({
       commands: [
-        {"type":"waitBlock","blocks":10},
-        {"type":"sendTransaction","account":3,"beneficiary":0,"eth":9}
+        { type: "waitTime","seconds":duration.days(1)},
+        { type:"sendTransaction","account":3,"beneficiary":0,"eth":9}
       ],
       crowdsale: {
         publicPresaleRate: 33, rate1: 18, rate2: 33, privatePresaleRate: 48,
-        foundationWallet: 1, setWeiLockBlocks: 1, owner: 7
+        foundationWallet: 1, setWeiLockBlocks: 600, owner: 7
       }
     });
 
     await runGeneratedCrowdsaleAndCommands({
       commands: [
-        {"type":"waitBlock","blocks":27},
-        {"type":"pauseCrowdsale","pause":true,"fromAccount":8},
-        {"type":"sendTransaction","account":0,"beneficiary":9,"eth":39}
+        { type: "waitTime","seconds":duration.days(2.6)},
+        { type:"pauseCrowdsale","pause":true,"fromAccount":8},
+        { type:"sendTransaction","account":0,"beneficiary":9,"eth":39}
       ],
       crowdsale: {
         publicPresaleRate: 1, rate1: 39, rate2: 13, privatePresaleRate: 35,
-        foundationWallet: 8, setWeiLockBlocks: 1, owner: 9
+        foundationWallet: 8, setWeiLockBlocks: 600, owner: 9
       }
     });
 
   });
 
-  it("calculates correct rate on the boundaries between endBlock1 and endBlock2", async function() {
+  it("calculates correct rate on the boundaries between endTime1 and endTime2", async function() {
     let crowdsaleAndCommands = {
       commands: [
-        { type: 'checkRate' },
-        { type: 'setWeiPerUSDinPresale', wei: 3000000000000000, fromAccount: 3 },
-        { type: 'checkRate' },
-        { type: 'waitBlock', blocks: 29 },
-        { type: 'buyTokens', beneficiary: 3, account: 2, eth: 12 }
+        { type: "checkRate" },
```

Comment:
why all the changes from single quote ' to double quotes "? Does your editor does it automatically? we have a big diff already to add all those :)

Reply:
I changed the all to use single quotes.

Reply:
I don't see them changed to single quotes

Reply:
I will remove the changes on this lines on the diff, I can do all that changes later.